libiocage/lib/Jail.py

```diff
@@ -511,7 +552,7 @@ def _launch_jail(self):
 
     def _start_vimage_network(self):
 
-        self.logger.log("Starting VNET/VIMAGE", jail=self)
+        self.logger.debug("Starting VNET/VIMAGE", jail=self)
```

Comment:
Why debug and not info?

Reply:
The library is not supposed to print. Info/Log gets printed by default and we want all stdout to be generated from the event responses.